src/reactive/value.rs

```diff
@@ -2927,6 +2928,15 @@ pub enum Value<T> {
     Dynamic(Dynamic<T>),
 }
 
+impl<T> PartialEq for Value<T>
+where
+    T: PartialEq + Clone
```

Comment:
This shouldn't require Clone, and I also don't like that `Dynamic`'s implementation of `PartialEq` is different than `Value::Dynamic`'s PartialEq result. To me PartialEq should work the same for a wrapped value as the wrapper in this situation.